.github/workflows/build.yml

```diff
@@ -50,10 +50,15 @@ jobs:
     - uses: actions/download-artifact@v2
       with:
         name: site
+    # for removing files, we need to start fresh; this does not remove dot-files
+    - run: rm -rf *
     - run: tar -xvf site.tar
```

Comment:
Won't this file have been deleted by the previous command?

Reply:
ahem, yes

Reply:
ok, have moved the `rm` further up